.github/workflows/upload_release_assets.yml

```diff
@@ -25,3 +25,18 @@ jobs:
           asset_path: ./assets/ipfix-collector.yaml
           asset_name: ipfix-collector.yaml
           asset_content_type: application/octet-stream
+  build-consumer:
```

Comment:
build-kafka-collector here and build-ipfix-collector above

Reply:
Sorry just notice this comment. Done. Thanks!